pkg/controllers/user/pipeline/controller/pipelineexecution/logsyncer.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	syncLogInterval = 10 * time.Second
+	syncLogInterval = 3 * time.Second
```

Comment:
Do we really need to sync in such short interval?

Reply:
This interval is for getting updates from Jenkins. The UX is bad when users need to wait for a long time to see any updates on state/log changes, I would love to get the UX right for now. It might cause overhead problem when simultaneously running plenty of pipelines, so stress tests or even refactor to write standalone websocket server to reduce etcd writes are being considered in the next phase.

Reply:
5 seconds sound better :)